samples/high_throughput_with_dask/echo.py

```diff
@@ -0,0 +1,6 @@
+import dask_espresso as de
+
+data = de.get_data_from_stdin()
+data.update(processed=True)
+
+print(de.encode_transport_data(data))
```

Comment:
Same as above, is the print statement intentional here?

Reply:
Yes. This script mocks a simulation for testing. I made the purpose clear in the doc string